JenkinsPods.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: golang
+spec:
+  containers:
+  - name: golang
+    image: golang:1.11.2-alpine
```

Comment:
why not use golang:1.12-alpine?

Reply:
I just used the one from here https://github.com/AirHelp/filler/blob/master/Dockerfile-test#L1